fdbserver/workloads/TaskBucketCorrectness.actor.cpp

```diff
@@ -43,7 +43,11 @@ struct SayHelloTaskFunc : TaskFuncBase {
 		// check task version
 		uint32_t taskVersion = task->getVersion();
 		if (taskVersion > SayHelloTaskFunc::version) {
-			TraceEvent("TaskBucketCorrectnessSayHello").detail("CheckTaskVersion", "taskVersion is larger than the funcVersion").detail("TaskVersion", taskVersion).detail("FuncVersion", SayHelloTaskFunc::version);
+			uint32_t v = SayHelloTaskFunc::version;
```

Comment:
Are we not allowed to pass this version in directly anymore?

Reply:
No: the `detail` function now takes a const reference. But you can't take a const reference of a constexpr field. I am actually not quite sure how this worked in the first place.

I could try to play some weird template tricks to make this work again. But in the whole code base I think this problem existed only once so I thought that adding more template code and increasing compilation time is not worth for this one occurrence.

Reply:
Ok, that seems fine to me.